tools/frr-reload.py

```diff
@@ -641,7 +642,7 @@ def load_contexts(self):
                 self.save_contexts(ctx_keys, cur_ctx_lines)
 
                 # exit current context
-                log.debug("LINE %-50s: exit context %-50s", line, ctx_keys)
+                log.debug("LINE {:<50}: exit context {:<50}".format(line, ' '.join(ctx_keys)))
```

Comment:
This works, but earlier you used format strings, why not stick with it?
```suggestion
                log.debug(f"LINE {line:<50}: exit context {' '.join(ctx_keys):<50}")
```